job.py

```diff
@@ -17,73 +16,52 @@ def __init__(self, url, email, password, catalogueDB, sourceDB):
         self.catalogueDB = catalogueDB
         self.sourceDB = sourceDB
 
-        ## Client serves as a general exm2 api client 
+        # Client serves as a general exm2 api client
         self.staging = Client(url=self.url, database=sourceDB, email=self.email, password=self.password)
         self.catalogue = Client(url=self.url, database=self.catalogueDB, email=self.email, password=self.password)
-        ## CatalogueClient serves client specific for use with the catalog model
+        # CatalogueClient serves client specific for use with the catalog model
         self.catalogueClient = CatalogueClient(self.staging, self.catalogue)
 
-    def sync(self):
-        """Sync staging with catalogue"""
-        self.cohortPid = self.__fetchCohortPid(self.staging, self.sourceDB)
-        if self.cohortPid is None:
-            log.info('Skip sync for: ' + self.sourceDB)
+        # Create logger
+        self.log = logging.getLogger(__name__)
```

Comment:
why a new logger per job instance ?

Reply:
only have to declare the logger once. but it is indeed better to have logging separate per job type. i will change this.